.github/workflows/overcloud-host-image-build.yml

```diff
@@ -94,11 +94,15 @@ jobs:
         if: inputs.centos
```

Comment:
hmm, on L35 it's using Kayobe stackhpc/wallaby

Reply:
I've swapped to Xena, for rocky support